client/coral-embed-api/index.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable no-var */
+/**
+ * @file Exposes the main 'window.Coral.Talk' API that developers can use to
```

Comment:
@jde I forgot what you told me to call this module so made something up. Remind me if you want me to change it.